config/initializers/sufia.rb

```diff
@@ -102,13 +104,9 @@
   # Specify the Fedora pid prefix:
   # config.id_namespace = "sufia"
 
-  
-  # Specify the path to the minter-state file
-  config.minter_statefile = "tmp/minter-state"
-
   # Specify the path to the file characterization tool:
   # config.fits_path = "fits.sh"
-  config.fits_path = "fits"
+   config.fits_path = "/usr/local/bin/fits-0.6.2/fits.sh"
```

Comment:
I think some unrelated changes got included here - minter-state and fits path should stay as they were, shouldn't they? I'm getting the old "Permission denied @ rb_sysopen - /tmp/minter-state" error.